### ISSUE TITLE
RadListView with swipeAction crash log

### ISSUE THREAD

**Issue by NickIliev** [telerik/nativescript-ui-feedback#863]
The error log is received via Crashlytics. Currently, there are no steps to reproduce the issue.

```
Fatal Exception: java.lang.NullPointerException
Attempt to read from field 'android.view.View android.support.v7.widget.RecyclerView$ViewHolder.itemView' on a null object reference
com.telerik.widget.list.SwipeActionsBehavior.createViewHolder (SwipeActionsBehavior.java:162)
com.telerik.widget.list.ListViewWrapperAdapter.onCreateListViewHolder (ListViewWrapperAdapter.java:459)
com.telerik.widget.list.ListViewAdapterBase.onCreateViewHolder (ListViewAdapterBase.java:145)
com.telerik.widget.list.ListViewAdapterBase.onCreateViewHolder (ListViewAdapterBase.java:22)
android.support.v7.widget.RecyclerView$Adapter.createViewHolder (RecyclerView.java:6685)
android.support.v7.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline (RecyclerView.java:5869)
android.support.v7.widget.GapWorker.prefetchPositionWithDeadline (GapWorker.java:285)
android.support.v7.widget.GapWorker.flushTaskWithDeadline (GapWorker.java:342)
android.support.v7.widget.GapWorker.flushTasksWithDeadline (GapWorker.java:358)
android.support.v7.widget.GapWorker.prefetch (GapWorker.java:365)
android.support.v7.widget.GapWorker.run (GapWorker.java:396)
android.os.Handler.handleCallback (Handler.java:739)
android.os.Handler.dispatchMessage (Handler.java:95)
android.os.Looper.loop (Looper.java:158)
android.app.ActivityThread.main (ActivityThread.java:7225)
java.lang.reflect.Method.invoke (Method.java)
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1230)
com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1120)

```

Reported via t.1345075

**Comment by zbranzov** [comment]
Without steps to reproduce we are unable to verify the fix. I hope it works. The next released version will include it. Stay tuned.

**Comment by zbranzov** [comment]
The fix is released in nativescript-ui-listview@3.8.0.
